_posts/2019-06-04-bitcoin-and-the-store-of-value-narrative.md

```diff
@@ -0,0 +1,69 @@
+
+In an attempt to dodge the Regression Theorem debate others have retorted with Nick Szabo’s views in his remarkable essay [“Shelling out: The Origins of Money”](https://nakamotoinstitute.org/shelling-out/), using the collectible and store of value arguments. Szabo’s excellent work is also a historical explanation like Menger, but more detailed and precise from an anthropological point of view.
+
+I shall not spend much time rebutting the claim that Bitcoin’s source of value at the very beginning was being a collectible in the literal meaning of the term. That was not what Szabo meant and it is not credible at all to say that the first Bitcoin owners demanded it because it was a whimsy, rather than demanding it because they thought it could potentially become a medium of exchange. Bitcoin’s first owners knew well why Bitcoin was invented and for what purpose it was intended. 
+
+
+A popular [narrative amongst Bitcoin supporters](https://medium.com/@vijayboyapati/the-bullish-case-for-bitcoin-6ecc8bdecc1) that apply Szabo’s views claim that money **always** evolves through the following stages:
+
+1. Collectible<br><br>
+2. Store of Value<br><br>
+3. Medium of exchange<br><br>
+4. Unit of account
+
+In my view, the above narrative is scientifically inconsistent and as such, it could be easily utilized against Bitcoin by academia and governments to spread negative propaganda like accusing Bitcoin of being a reckless speculative bubble and that there is no scientific support of its value other than “It’s valuable because people value it”. It’s not that I fear too much that they can stop anything - I believe Bitcoin will thrive or not regardless of what haters and supporters claim - though, they might be able to slow down the adoption process a bit.
+
+
+The narrative is scientifically inconsistent because it deviates from the [Subjective Theory of Value](https://en.wikipedia.org/wiki/Subjective_theory_of_value) as it puts value in front of utility. I don’t want to claim that this theory is set in stone but, if it has to be amended, it has to be very well justified. In my view, there are absolutely no grounds for amendment within the above narrative. As I fully reject the collectible whimsy argument to explain Bitcoin bootstrapping, the only argument left is that the collectible is valuable because it is a store of value, which is obviously circular and therefore explains nothing. Game theory does not resolve that circularity either. Moreover, it is also confusing that the narrative assigns to store of value the properties which belong to medium of exchange (portable, fungible, divisible, quickly verifiable, etc).  
+
+
+Here I will explain how Bitcoin fits very well within the Subjective Theory of Value and also within the historical stages of exchange described by Carl Menger. Leaving apart deferred exchanges (i.e. credit), the basic first three stages of spot exchange are the following:
+
+**Direct exchange**: Barter.
```

Comment:
When you view this section in the browser, these items are all melded on to one line.